glfw31-gl41core-triangle/triangle.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2014 The go-gl Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Renders a simple static triangle using GLFW 3.1 and OpenGL 4.1 core forward-compatible profile.
+package main
+
+import (
+	"fmt"
+	"runtime"
+
+	gl "github.com/go-gl/gl/v4.1-core/gl"
+	glfw "github.com/go-gl/glfw/v3.1/glfw"
```

Comment:
No need for `gl` and `glfw` prefixes here. The last path element gives that away:

```
	"github.com/go-gl/gl/v4.1-core/gl"
	"github.com/go-gl/glfw/v3.1/glfw"
```